pkg/scheduler/framework/plugins/nodelabel/node_label_test.go

```diff
@@ -73,69 +73,87 @@ func TestValidateNodeLabelArgs(t *testing.T) {
 func TestNodeLabelFilter(t *testing.T) {
 	label := map[string]string{"foo": "any value", "bar": "any value"}
 	var pod *v1.Pod
+	node := v1.Node{ObjectMeta: metav1.ObjectMeta{Labels: label}}
 	tests := []struct {
-		name    string
-		rawArgs string
-		res     framework.Code
+		name      string
+		nodeExist bool
```

Comment:
I wouldn't complicate all the unit tests for this case. Just create a separate, much simpler test.